Makefile

```diff
@@ -19,6 +19,7 @@ export PYTHONPATH = $(shell echo "$$PYTHONPATH"):$(shell python -c 'import os; p
 all: clean-pyc check test
 
 check:
+	@$(PYTHON) scripts/check_crlf.py pygments dist || true
```

Comment:
why `dist`? It contains only tarballs/wheels...

Reply:
Good catch, that should have been build. I've also added `external`.